front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/CommunityMonitorCTA/index.tsx

```diff
@@ -24,14 +24,20 @@ const CommunityMonitorCTA = () => {
   // TEMPORARY: While waiting for final designs.
   return (
     <Box my="20px">
+      <Title textAlign="center" color="grey500" variant="h2">
+        {'----- [PLACEHOLDER] -----'}
+      </Title>
       <Title textAlign="center" variant="h3">
-        How do you feel about living in your city?
+        How do you feel about living in Gothenburg?
       </Title>
       <Box display="flex" justifyContent="center">
```

Comment:
You can ignore this file for now, since I'm merging this PR into a WIP PR for the Homepage Builder Widget.